frontend/src/components/Year/Year.tsx

```diff
@@ -15,7 +15,12 @@ import { AppState } from "../../state/reducers/state";
 import { connect } from "react-redux";
 import { Dispatch } from "redux";
 import { toggleYearExpandedForActivePlanAction } from "../../state/actions/userPlansActions";
-import { ScheduleCourse } from "../../../../common/types";
+import {
+  ScheduleCourse,
+  SeasonWord,
+  StatusEnum,
+} from "../../../../common/types";
+import SemesterContainer from "../SemesterContainer";
```

Comment:
Totally not part of this ticket but it would be reallyy nice if we could turn this into a functional component 🥺 since we're trying to move towards those as and when we touch legacy class components